lib/Resque.pm

```diff
@@ -97,7 +101,7 @@ has redis => (
     is      => 'ro',
     lazy    => 1,
     coerce  => 1,
-    isa     => 'Sugar::Redis',
+    isa     => 'Sugar::Redis|Object',
```

Comment:
Ah, I see that the standard pure perl Redis module doesn't actually implement the Redis commands directly, only via AUTOLOAD, so 'can' doesn't really work, so duck_type is going to be tricky, and not worth the effort of hacking in, after all.

Cheers!